task-5/calculator/script.js

```diff
@@ -0,0 +1,303 @@
+
+var expression;
+var active;
+
+
+var expressionTxt = '';
+var activeNUmber = '0';
+
+var isAfterOperation = false;
+var isAfterExecute = false;
+
+var prevActiveNumber = null;
+var prevLastoperation = null;
+var lastoperation = null;
+
+window.onload = function () {
+
+    expression = document.getElementById('expression');
+    active = document.getElementById('active');
+
+    const button0 = document.getElementById('0');
+    const button1 = document.getElementById('1');
+    const button2 = document.getElementById('2');
+    const button3 = document.getElementById('3');
+    const button4 = document.getElementById('4');
+    const button5 = document.getElementById('5');
+    const button6 = document.getElementById('6');
+    const button7 = document.getElementById('7');
+    const button8 = document.getElementById('8');
+    const button9 = document.getElementById('9');
+
+    const buttonDiv = document.getElementById('Div');
+    const buttonMult = document.getElementById('Mult');
+    const buttonSub = document.getElementById('Sub');
+    const buttonAdd = document.getElementById('Add');
+    const buttonRoot = document.getElementById('Root');
+    const buttonDeg = document.getElementById('Deg');
+
+    const buttonReset = document.getElementById('Reset');
+    const buttonExecute = document.getElementById('Execute');
+    const buttonSlice = document.getElementById('Slice');
+
+    active.innerHTML = activeNUmber;
+
+    button0.addEventListener("click", () => { inputDigital(0); });
+    button1.addEventListener("click", () => { inputDigital(1); });
+    button2.addEventListener("click", () => { inputDigital(2); });
+    button3.addEventListener("click", () => { inputDigital(3); });
+    button4.addEventListener("click", () => { inputDigital(4); });
+    button5.addEventListener("click", () => { inputDigital(5); });
+    button6.addEventListener("click", () => { inputDigital(6); });
+    button7.addEventListener("click", () => { inputDigital(7); });
+    button8.addEventListener("click", () => { inputDigital(8); });
+    button9.addEventListener("click", () => { inputDigital(9); });
+
+    buttonDiv.addEventListener("click", () => { inputOperation(1); });
+    buttonMult.addEventListener("click", () => { inputOperation(2); });
+    buttonSub.addEventListener("click", () => { inputOperation(3); });
+    buttonAdd.addEventListener("click", () => { inputOperation(4); });
+    buttonRoot.addEventListener("click", () => { inputOperation(5); });
+    buttonDeg.addEventListener("click", () => { inputOperation(6); });
+
+    buttonReset.addEventListener("click", () => { resetAll(); });
+    buttonExecute.addEventListener("click", () => { executeExpression(); });
+    buttonSlice.addEventListener("click", () => { sliceActiveNumb(); });
```

Comment:
it might be simplified

`buttonSlice.addEventListener("click", () => { sliceActiveNumb(); });`
->
`buttonSlice.addEventListener("click", sliceActiveNumb);`